news/2022-04-13-new-featured-artist-in-love-with-a-ghost.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "New Featured Artist: in love with a ghost"
+date: 2022-04-13 09:00:00 +0000
+---
+
+Take a break with our newest Featured Artist **in love with a ghost**.
+
+![](https://assets.ppy.sh/artists/257/header.jpg)
+
+Morning brightness shines through the blinds into your bedroom. You know you have to rise at some point, but the warmth of your comforter is far too enticing. You lay fully satisfied for just a minute longer before starting your day. Congratulations, you've just experienced your first **in love with a ghost** track.
+
+French artist **in love with a ghost** is the last thing you'd expect to find in a rhythm game. Their songs are mainly gentle piano pieces with touches of jazz and other calm styles, yet despite being so far from the norm, their music struck has assuredly struck chord with osu! players.
+
+**10** relaxing tunes. All available from [**in love with a ghost**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/257). Soak it all in.
+
+In case you've never done late night homework assignments while listening to **in love with a ghost** on loop, explore this quick video of [interdimensional portal leading to a cute place](https://osu.ppy.sh/beatmapsets/1729796) and the rest of our previews:
```

Comment:
yes it's a name of a track but it doesn't hurt to incorporate it into the sentence

```suggestion
In case you've never done late-night homework assignments while listening to **in love with a ghost** on loop, explore this quick video of an [interdimensional portal leading to a cute place](https://osu.ppy.sh/beatmapsets/1729796) and the rest of our previews:
```